user/languages/ruby.md

```diff
@@ -7,104 +7,62 @@ layout: en
 <div id="toc">
 </div>
 
-## What This Guide Covers
-
-This guide covers build environment and configuration topics specific to Ruby
-projects. Please make sure to read our [Getting
-Started](/user/getting-started/) and [general build
-configuration](/user/customizing-the-build/) guides first.
-
-## Supported Ruby Versions and RVM
-
-The Ruby VM provides recent versions of:
-
-- Ruby: 2.2.0, 2.1.x, 2.0.0, 1.9.3, 1.9.2 and 1.8.7
-- JRuby: 1.7.x (1.8 and 1.9 mode)
-- Ruby Enterprise Edition: 1.8.7 2012.02
-
-Pre-compiled versions are downloaded on demand from our [Ruby version cache](http://rubies.travis-ci.org).
+> We're moving all version numbers and other distro specific information over to
```

Comment:
How about: 

> Language versions and other build-environment specific
> information are in our reference pages:
>  * [Precise](/user/reference/precise/)
>  * [Trusty](/user/reference/trusty/)
>  * [OS X](/user/reference/osx/)

The rest of this guide covers configuring Ruby projects on Travis CI. If you're
new to Travis CI please read our [Getting Started](/user/getting-started/) and
[build configuration](/user/customizing-the-build/) guides first.